### PR TITLE
[stable/fluentd-elasticsearch] Adding support for the logstash_prefix option

### PR DESCRIPTION
**Pull request by cidrick**
Signed-off-by: Matt Renfrow <mrenfrow@expedia.com>

#### What this PR does / why we need it:

This PR adds support for setting the `logstash_prefix` option for the Elasticsearch output plugin for fluentd. Without being able to set this option, all logs will be sent to an index pattern of `logstash-YYMMDD`, which is undesirable when using a mixed-use Elasticsearch environment.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by unguiculus** [comment]
cc @monotek

**Comment by cidrick** [comment]
@unguiculus I believe this is just waiting for your sign-off :)

**Review by paulczar — CHANGES_REQUESTED** [review]
to pass tests we need the volume name here to be shortened ...   would do that in the template helpers.

```
Error: release fluentd-elasticsearch-ff5f0rdix3 failed: DaemonSet.apps "fluentd-elasticsearch-ff5f0rdix3-fluentd-elasticsearch" is invalid: [spec.template.spec.volumes[3].name: Invalid value: "config-volume-fluentd-elasticsearch-ff5f0rdix3-fluentd-elasticsearch": must be no more than 63 characters, spec.template.spec.containers[0].volumeMounts[3].name: Not found: "config-volume-fluentd-elasticsearch-ff5f0rdix3-fluentd-elasticsearch"]
```

**Pull request merged**
